src/transformState.js

```diff
@@ -6,6 +6,28 @@
  */
 function transformState(state, actions) {
   // write code here
+
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+
+      case 'removeProperties':
+        for (const key in state) {
+          if (actions[i].keysToRemove.includes(key)) {
+            delete state[key];
+          }
```

Comment:
you don't need this if here, if key doesn't exist in the object then delete will ignore this key